eZ/Publish/Core/REST/Server/Input/Parser/SortClause/GenericDataKey.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace eZ\Publish\Core\REST\Server\Input\Parser\SortClause;
+
+use eZ\Publish\API\Repository\Values\Content\Query;
+use eZ\Publish\Core\REST\Common\Input\BaseParser;
+use eZ\Publish\Core\REST\Common\Input\ParsingDispatcher;
+use eZ\Publish\Core\REST\Common\Exceptions;
+
+class GenericDataKey extends BaseParser
```

Comment:
I'd remove "generic", but include the fact that it uses a value object class.

`DataKeyValueObjectClass` ? A bit verbose, but at least it is descriptive :-)